CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,128 @@
+# Contributor Covenant Code of Conduct
```

Comment:
Did you guys discuss the move from https://github.com/bvaughn/react-error-boundary/blob/master/other/CODE_OF_CONDUCT.md  (http://contributor-covenant.org/version/1/4) to this version (https://www.contributor-covenant.org/version/2/0/)? The commit message makes it sound like the change was accidental...

Reply:
I've been slowly upgrading all of my OSS projects to the latest version.